test/bruno/Notifications-v2/environments/tt02.bru

```diff
@@ -0,0 +1,11 @@
+vars {
+  host: https://platform.{{Environment}}.altinn.no
+  Environment: tt02
+}
+vars:secret [
+  TokenGeneratorBaseUrl,
+  jwt,
+  TokenGeneratorUserName,
```

Comment:
I have added these as env-variables in a .env file for profile, as this is how it's handled in authorizations bruno collection 
(https://github.com/Altinn/altinn-profile/pull/374/files#diff-da40235a1604206c59bfaa65373933ace383c5106ef9b2db4d89be222efe54ca )

Reply:
I think @eskebab considered this and landed on the built-in sectrets in order to be able to swap environments... 

A side-note is that I also have an idea for a PR to Bruno itself to permit .env-files with some sort of hirearchical structure (if it is not already possible - right now it seems you need to juggle multiple env-files and remember to not check them in... https://[docs.usebruno.com/secrets-management/dotenv-file](https://docs.usebruno.com/secrets-management/dotenv-file)  ... or ofcourse pay up and use a proper secrets manager :-) )

Reply:
The "remember to not check them in" part is easy with a `.gitignore` ([example](https://github.com/Altinn/altinn-profile/pull/374/files#diff-d6f4c0f63b0e520bb1878fdb29166ef22027a195ad4e4ee38df7ee82a4787b26)).